extensions/common/BUILD.gn

```diff
@@ -0,0 +1,10 @@
+source_set("common") {
+  sources = [
+    "brave_extension_urls.cc",
+    "brave_extension_urls.h",
+  ]
+
+  deps = [
+    "//base",
```

Comment:
also just for reference this set of deps is not complete. It should also have `//extensions/common` and `//url` - you won't actually need this anymore, but just for future reference you need a dep for anything in the headers

Reply:
`//url` - https://github.com/brave/brave-core/pull/2779/files#diff-13ddefeda1421d0383faa29f42096c79R8
`//extensions/common` - https://github.com/brave/brave-core/pull/2779/files#diff-13ddefeda1421d0383faa29f42096c79R7